armtemplates/alerts/azuredeploy_Memory.json

```diff
@@ -65,8 +65,9 @@
       "properties": {
         "mode": "Incremental",
         "Category": "AlertIbiza",
-        "DisplayName": "CPU Usage Alert",
-        "Query": "Perf\n| where TimeGenerated > ago(1m) \n| where CounterName == \"% Processor Time\" and InstanceName == \"_Total\"\n| summarize AggregatedValue = avg(CounterValue) by Computer\n| where AggregatedValue > 90",
+        "DisplayName": "Memory Usage Alert",
+        "Query": "Perf\n| where TimeGenerated > ago(1m)\n| where CounterName == \"% Processor Time\" and InstanceName == \"_Total\"\n| summarize AggregatedValue = avg(CounterValue) by Computer\n| where AggregatedValue > 90",
```

Comment:
Should this one still be > 90%? Or will we use a different metric?

Reply:
Pretty sure this one is actually OK